src/bttrx_control.cpp

```diff
@@ -161,6 +168,9 @@ ResultType BTTRX_CONTROL::action(string name)
 void BTTRX_CONTROL::storeSetting(ParameterType type, string value)
 {
 	switch (type) {
+	case kCallsign:
+		callsign_ = value;
+		break;
```

Comment:
Please remove, as this is not necessary.
This method is only for data which is stored as a setting inside the wt32i module.
The callsign is stored as a Preference in the ESP32.

Reply:
Solved with https://github.com/phl0/firmware/commit/9bece9392f208d0897e485a552e37f3b87ffc408. (Will rebase to 1.1.0 dev branch as soon as all issues are done).